cmd/drand-cli/control.go

```diff
@@ -138,7 +145,7 @@ func leadShareCmd(c *cli.Context) error {
 		"file will not be written out to the specified output. To get the "+
 		"group file once the setup phase is done, you can run the `drand show "+
 		"group` command")
-	groupP, shareErr := ctrlClient.InitDKGLeader(nodes, args.threshold, period, args.timeout, args.entropy, args.secret, offset)
+	groupP, shareErr := ctrlClient.InitDKGLeader(nodes, args.threshold, period, catchupPeriod, args.timeout, args.entropy, args.secret, offset)
```

Comment:
That has nothing to do with the randomness generation I would avoid putting more args to the DKG. 
We should thirve towards even removing the `period` call.

Reply:
this is how the command gets from the cli to the daemon, right?

Reply:
That's right ... I just dislike the idea to putting more and more stuff related to beacon chain to DKG code. But I guess it'll have to do for this time.